### PR TITLE
Fixes and simplifies escape shuttle timers

### LINKED ISSUE #7033: Text on the status display set from the communications console doesn't appear again after the shuttle is called and recalled.
### Environment
- Game Version: Production 4119
- OS Version: Windows 10

### Detailed description of bug
When text is set on status displays via the communications console and the shuttle is called and recalled, text that was set by the comms console will not appear again even if new text is set.

### Steps to reproduce
1. As a member of command, use the communications console to set text on the status displays.
2. Check the status displays to make sure they aren't blank.
3. Call the shuttle. Recall it.
4. Check the status displays, the text will be gone.
5. Use the communications console to set text on the status displays again.
6. Check the status displays again, they will still be blank.

### PR DESCRIPTION
**Pull request by Aranclanos**
### Purpose
Simplifies escape shuttle code reducing the amount of event invokes and functions inside functions.
CL: [Fix] Fixes shuttles not being able to be recalled.
CL: [Fix] Fixes the escape shuttle time being wrong on code blue.
CL: [Fix] Fixes status display not returning to its original channel once the shuttle is recalled.
CL: [Fix] Fixes status display not showing the recall timer.
Fixes #7033

### CODE REVIEW

File: UnityProject/Assets/Scripts/Managers/CentComm.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using AddressableReferences;
+using Google.Protobuf.WellKnownTypes;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Managers/CentComm.cs]: Nani

File: UnityProject/Assets/Scripts/Managers/GameManager.Escape.cs [hunk 2]
```diff
@@ -101,50 +103,39 @@ private void InitEscapeStuff()
...
-			{
-				beenToStation = true;
-				_ = SoundManager.PlayNetworked(CommonSounds.Instance.ShuttleDocked);
-				Chat.AddSystemMsgToChat(string.Format(ChatTemplates.PriorityAnnouncement, $"<color=white>Escape shuttle has arrived! The shuttle <color=#FF151F>cannot</color> leave the station due to the hostile environment!</color>"), MatrixManager.MainStationMatrix);
-			}
-		} );
+	private IEnumerator WaitForRoundEnd()
+	{
+		Logger.Log($"Shuttle docked to Centcom, Round will end in {TimeSpan.FromSeconds(RoundEndTime).Minutes} minute", Category.Round);
+		yield return WaitFor.Seconds(1f);
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Managers/GameManager.Escape.cs:137]: 1 second to end the round?

File: UnityProject/Assets/Scripts/Managers/CentComm.cs (hunk 1 above)
Comment (Aranclanos) [reply UnityProject/Assets/Scripts/Managers/CentComm.cs]: @ rider why did you add this

File: UnityProject/Assets/Scripts/Managers/GameManager.Escape.cs (hunk 2 above)
Comment (Aranclanos) [reply UnityProject/Assets/Scripts/Managers/GameManager.Escape.cs:137]: I only moved this but apparently there's a second waitfor with the real timer

**Review by Bod9001 — DISMISSED** [review]

**Review by Bod9001 — APPROVED** [review]

**Review by corp-0 — APPROVED** [review]

**Pull request merged**